gems/aws-sdk-core/spec/spec_helper.rb

```diff
@@ -70,6 +70,16 @@ def client_with_plugin(options = {}, &block)
       client_class.new(options)
     end
 
+    # clears opentelemetry-sdk configuration state between specs
+    # https://github.com/open-telemetry/opentelemetry-ruby/blob/main/test_helpers/lib/opentelemetry/test_helpers.rb#L18
+    def reset_opentelemetry_sdk
```

Comment:
You can possibly just move this to an after {} in shared_spec_helper and have it run all the time.

Reply:
I think I'll opt to having this method here to use when I need it.